### PR TITLE
User profile page closes #23

### LINKED ISSUE #23: As a registered user I can see and modify my account details.

### PR DESCRIPTION
**Pull request by paulgrever**

### CODE REVIEW

File: app/assets/stylesheets/custom.css.scss [hunk 1]
```diff
@@ -105,6 +105,15 @@ li { list-style: none; }
   background-color: $site-brown;
 }
 
+#sign-in { margin: 50px; }
```
Comment (paulgrever) [inline app/assets/stylesheets/custom.css.scss]: This is a one time item

File: app/assets/stylesheets/custom.css.scss [hunk 2]
```diff
@@ -105,6 +105,15 @@ li { list-style: none; }
...
+
+#supply-hist {
+  margin-top: 0px;
```
Comment (paulgrever) [inline app/assets/stylesheets/custom.css.scss]: resolved

File: spec/features/user/user_show_spec.rb [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      expect(page).to have_content("Laul Guberson")
+    end
+  end
+
+  context "after logging in"do
+    before  :each do 
+      @user = FactoryGirl.create(:user)
+      item = FactoryGirl.create(:item)
+      order = FactoryGirl.create(:order, user_id: @user.id)
+      order_item = FactoryGirl.create(:order_item, item_id: item.id, order_id: order.id)
```
Comment (paulgrever) [inline spec/features/user/user_show_spec.rb]: Order is needed to reference user to item

File: app/assets/stylesheets/custom.css.scss [hunk 4]
```diff
@@ -105,6 +105,15 @@ li { list-style: none; }
   background-color: $site-brown;
 }
 
+#sign-in { margin: 50px; }
+
+#supply-hist {
```
Comment (paulgrever) [inline app/assets/stylesheets/custom.css.scss]: No

File: spec/features/user/user_show_spec.rb [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+  context "after logging in"do
+    before :each do
+      @user = FactoryGirl.create(:user)
+      item = FactoryGirl.create(:item)
+      order = FactoryGirl.create(:order, user_id: @user.id)
+      order_item = FactoryGirl.create(:order_item,
+                                      item_id: item.id,
+                                      order_id: order.id)
+      order = FactoryGirl.create(:order, user_id: @user.id)
```
Comment (paulgrever) [inline spec/features/user/user_show_spec.rb]: You must create an order to access items from user.

File: spec/features/user/user_show_spec.rb [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+      expect(page).to have_content("Laul Guberson")
+    end
+  end
+
+  context "after logging in"do
+    before :each do
+      @user = FactoryGirl.create(:user)
+      item = FactoryGirl.create(:item)
+      order = FactoryGirl.create(:order, user_id: @user.id)
+      order_item = FactoryGirl.create(:order_item,
```
Comment (paulgrever) [inline spec/features/user/user_show_spec.rb]: You must create an order to access items from user.

File: spec/models/user_spec.rb [hunk 7]
```diff
@@ -1,24 +1,32 @@
...
-  it 'should have many items' do
+  it "should have many items" do
     expect(@user).to respond_to(:items)
   end
+
+  it "it displays placed orders for user" do
+    item = FactoryGirl.create(:item)
+    order = FactoryGirl.create(:order, user_id: @user.id)
+    order_item = FactoryGirl.create(:order_item, item_id: item.id, order_id: order.id)
+    order = FactoryGirl.create(:order, user_id: @user.id)
```
Comment (paulgrever) [inline spec/models/user_spec.rb]: You must create an order to access items from user.

File: spec/models/user_spec.rb [hunk 8]
```diff
@@ -1,24 +1,34 @@
...
     expect(@user).to respond_to(:items)
   end
+
+  it "it displays placed orders for user" do
+    item = FactoryGirl.create(:item)
+    order = FactoryGirl.create(:order, user_id: @user.id)
+    order_item = FactoryGirl.create(:order_item,
+                                    item_id: item.id,
+                                    order_id: order.id)
+    order = FactoryGirl.create(:order, user_id: @user.id)
```
Comment (paulgrever) [inline spec/models/user_spec.rb]: You must create an order to access items from user.

**Comment by paulgrever** [comment]
@indiesquidge @kpearson ready to be reviewed. Only hound issues are mistakes and it's hate for id selectors. All tests pass except single skip for the email validation - waiting on team decision. This implements user profile pages.

**Comment by kpearson** [comment]
Looks good!

File: spec/models/user_spec.rb (hunk 8 above)
Comment (indiesquidge) [inline spec/models/user_spec.rb]: Yeah but you don't have to assign it to a variable if it's not used elsewhere.

File: spec/features/user/user_show_spec.rb (hunk 5 above)
Comment (indiesquidge) [inline spec/features/user/user_show_spec.rb]: Yeah but you don't have to assign it to a variable if it's not used elsewhere.

File: spec/features/user/user_show_spec.rb (hunk 6 above)
Comment (indiesquidge) [inline spec/features/user/user_show_spec.rb]: Yeah but you don't have to assign it to a variable if it's not used elsewhere.

**Pull request merged**